package.json

```diff
@@ -24,8 +24,8 @@
     "posttest": "./bin/posttest.sh",
     "prettier": "prettier --write \"./packages/melody-*/src/**/*.[tj]s\"",
     "release": "./bin/release.sh",
-    "test": "jest",
-    "test-travis": "jest --maxWorkers=4",
+    "test": "TZ=Europe/London jest",
```

Comment:
I think it would be better to set it through a setup file something like `testsetup/timezone.js`.

Reply:
Okay. I'll do that.

Reply:
@efegurkan I need your help. I tried to set up` testsetup/timezone.js` using jest's `setupFiles ` option but it did not work.

I tried with 
```
process.env.TZ = 'Europe/London'
```
Also, I tried few approaches to setup `TZ` flag in node but unable to get a working solution.

I found few options to solve this issue by mocking Date with `moment.js` but I personally think that including such a big library is not a good choice.

Please let me know your thoughts.

Reply:
I'll try to figure it out, It is possible that because of Jest's runtime does not work as we expect it to work. 

Thanks for your efforts :)

Reply:
I checked the problem and don't think it is worth the effort for now. I can accept this but might change in future.

>I found few options to solve this issue by mocking Date with moment.js but I personally think that including such a big library is not a good choice.

Since it is a `devDependency` it is actually OK IMHO.

Reply:
I checked the problem and don't think it is worth the effort for now. I can accept this but might change in future.

>I found few options to solve this issue by mocking Date with moment.js but I personally think that including such a big library is not a good choice.

Since it is a `devDependency` it is actually OK IMHO.